components/lcd/esp_lcd_ssd1681/examples/epaper_lvgl_demo/README.md

```diff
@@ -0,0 +1,91 @@
+| Supported Targets | ESP32 | ESP32-C2 | ESP32-C3 | ESP32-C6 | ESP32-H2 | ESP32-S2 | ESP32-S3 |
+| ----------------- | ----- | -------- | -------- | -------- | -------- | -------- | -------- |
```

Comment:
I think we don't have to create this markdown table for a readme file out side of the esp-idf. User may be confusing about why this driver is not supported on a new esp target